src/test/java/hudson/plugins/parameterizedtrigger/test/BuildInfoExporterTest.java

```diff
@@ -175,7 +177,7 @@ public void test_multipletriggers() throws Exception {
                 new TriggerBuilder(
                 new BlockableBuildTriggerConfig(testNameResult + "," + testNameResult2,
                 new BlockingBehaviour(Result.FAILURE, Result.UNSTABLE, Result.FAILURE),
-                ImmutableList.of(new CounterBuildParameterFactory("0", Integer.toString(buildsToTest - 1), "1", "TEST=COUNT$COUNT")),
+                Arrays.asList(new CounterBuildParameterFactory("0", Integer.toString(buildsToTest - 1), "1", "TEST=COUNT$COUNT")),
```

Comment:
Collections.singletonList? as it's single entry List. but really nitpicking as it is a test class

Reply:
This is not a singleton

Reply:
Is this a singleton.  Code continues to the next line

Reply:
addressed in 2b76ed0